scripts/vnet_route_check.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
```

Comment:
please add the design behind this vnet route check.
what is the purpose, the flow the underlay requirements, etc.